File: 0392-is-subsequence/0392-is-subsequence.cpp
class Solution {
public:
    bool isSubsequence(string s, string t) {
        int n = s.length();
        int count = 0;
        int j = 0;
        for(int i = 0; i < t.size(); i++){
            if(t[i] == s[j]){
                count++;
                j++;
            }
        }
        return count == n;
    }
};

File: 0392-is-subsequence/0392-is-subsequence.py
class Solution(object):
    def isSubsequence(self, s, t):
        if not s:
            return True
        count = 0
        j = 0
        for i in t:
            if i == s[j]:
                count+=1
                j+=1
            if count == len(s):
                return True
        return False
        